MobileBuy/sample/build.gradle

```diff
@@ -39,12 +39,14 @@ android {
       def apiKey
       def androidPayPublicKey
       def androidPayEnvironment
+      def defaultLocale
 
       // get the shop setting from the environment if there any
       shopDomain = System.getenv("SHOP_DOMAIN")
       apiKey = System.getenv("API_KEY")
       androidPayPublicKey = System.getenv("ANDROID_PAY_PUBLIC_KEY")
       androidPayEnvironment = System.getenv("ANDROID_PAY_ENVIRONMENT")
+      defaultLocale = System.getenv("DEFAULT_LOCALE") ?: ""
```

Comment:
Is this set by macOS or is it more universal? Should this be explained somewhere?

Reply:
It should be okay.  It's a change in the sample app to make it simpler to change the default locale (especially for test purposes).